sdk/python/tests/compiler/compiler_tests.py

```diff
@@ -226,18 +226,7 @@ def _test_pipeline_workflow(self,
     golden_yaml_file = os.path.join(test_data_dir, pipeline_yaml)
     temp_dir = tempfile.mkdtemp()
     compiled_yaml_file = os.path.join(temp_dir, 'workflow.yaml')
-    try:
-      compiler.TektonCompiler().compile(pipeline_function,
-                                        compiled_yaml_file,
-                                        generate_pipelinerun=generate_pipelinerun,
-                                        enable_artifacts=enable_artifacts)
-      with open(compiled_yaml_file, 'r') as f:
-        f = normalize_compiler_output_function(
-          f.read()) if normalize_compiler_output_function else f
-        compiled = list(yaml.safe_load_all(f))
-      if GENERATE_GOLDEN_YAML:
-        with open(golden_yaml_file, 'w') as f:
-          f.write(textwrap.dedent("""\
+    license_header = textwrap.dedent("""\
```

Comment:
I meant move `license_header` to the top of the `compiler_tests.py` file since Andrew will have to use that in his PR as well and it blows up the code too much here

Reply:
oh you mean set it as global variable?